docs/CHANGELOG_UI_CALLING.md

```diff
@@ -1,10 +1,7 @@
 # Azure Communication UI Calling Release History
 
 ## 1.1.0 (2022-11-09)
-- This version is the public GA release with Calling UI Library
-
-### Bug Fixes
-- Error banner color for Dark theme corrected
+- This update includes the fix for Error banner and banner text color for dark theme
```

Comment:
Should we list here all features and bug fixes added to 1.1.0 in comparison with previous GA version 1.0.0?

Reply:
We already have them listed in the 1.1.0-beta.1 changeless